scripts/steps/build-api.js

```diff
@@ -110,6 +104,105 @@ const UNWRAP_UTILITIES = new Set([
   'Readonly'
 ])
 
+const EXCLUDED_SUBMENU_KINDS = new Set(['Type alias', 'Interface', 'Namespace'])
+const EXCLUDED_TREE_KINDS = new Set(['Namespace'])
+const CORE_ENTITIES = new Set([
+  'BaseNode',
+  'ClientNode',
+  'Connection',
+  'Log',
+  'LogStore',
+  'Reconnect',
+  'ServerConnection',
+  'ServerNode'
+])
+const CLIENT_ENTITIES = new Set([
+  'Client',
+  'CrossTabClient',
+  'badge',
+  'buildNewSyncMap',
+  'changeSyncMap',
+  'changeSyncMapById',
+  'confirm',
+  'createFilter',
+  'createSyncMap',
+  'defineSyncMap',
+  'deleteSyncMap',
+  'deleteSyncMapById',
+  'encryptActions',
+  'request'
+])
+const SERVER_ENTITIES = new Set([
+  'BaseServer',
```

Comment:
Move `BaseServer` to Others, since it is internal type. User should work with `Server` or `TestServer`.